Marlin/src/feature/leds/neopixel.cpp

```diff
@@ -90,35 +92,25 @@ void Marlin_NeoPixel::init() {
     safe_delay(500);
     set_color_startup(adaneo1.Color(0, 0, 255, 0));  // blue
     safe_delay(500);
-  #endif
+    #if HAS_WHITE_LED
+      set_color_startup(adaneo1.Color(0, 0, 0, 255));  // white
+      safe_delay(500);
+    #endif
 
-  #ifdef NEOPIXEL_BKGD_LED_INDEX
-    set_color_background();
   #endif
 
-  #if ENABLED(LED_USER_PRESET_STARTUP)
-    set_color(adaneo1.Color(LED_USER_PRESET_RED, LED_USER_PRESET_GREEN, LED_USER_PRESET_BLUE, LED_USER_PRESET_WHITE));
-  #else
-    set_color(adaneo1.Color(0, 0, 0, 0));
+  #ifdef NEOPIXEL_BKGD_INDEX_FIRST
+    set_color_background();
   #endif
-}
 
-#if 0
-bool Marlin_NeoPixel::set_led_color(const uint8_t r, const uint8_t g, const uint8_t b, const uint8_t w, const uint8_t p) {
-  const uint32_t color = adaneo1.Color(r, g, b, w);
-  set_brightness(p);
-  #if DISABLED(NEOPIXEL_IS_SEQUENTIAL)
-    set_color(color);
-    return false;
-  #else
-    static uint16_t nextLed = 0;
-    set_pixel_color(nextLed, color);
-    show();
-    if (++nextLed >= pixels()) nextLed = 0;
-    return true;
-  #endif
```

Comment:
We seem to have lost all the `NEOPIXEL_IS_SEQUENTIAL` functionality. For the new backlight range, this routine could be restored, but making sure to skip that whole set of indexes, if they are in the middle.

Reply:
this code is/was in a #if 0 block. has been disabled for some time...   It probably should be re-enabled or the option removed...

Reply:
Let me put back what I think is missing and after that you can let me know if you agree it is better to have it restored…

Reply:
Ha, yes I see that `set_led_color` was defined but nothing is using it. So far I don't see any instances of missing `isSequence`, so we might be good to go. Plodding on, I'll finish my review and get this baby merged. Great work!

Reply:
Ok… The one place I would want to put `isSequential` back is in the `pel_set_rgb` call to `leds.set_color` since Printer Event LEDs are the _raison d'être_ for sequential LEDs in the first place. Other than that everything is fab.